TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -204,7 +204,11 @@ def getTransformationDirectories(self, transID):
       transDirectories = []
       if res['Value']:
         if not isinstance(res['Value'], list):
-          transDirectories = ast.literal_eval(res['Value'])
+          try:
+            transDirectories = ast.literal_eval(res['Value'])
+          except:
+            # It can happen if the res['Value'] is '/a/b/c' instead of '["/a/b/c"]'
+            transDirectories = list(res['Value'])
```

Comment:
Hi,

Won't this result in transDirectories being set to ['/', 'a', '/', 'b', '/', 'c'] for the given example? Maybe it'd be safer to do `transDirectories = [ res['Value'] ]`?

Regards,
Simon

Reply:
Thank you Simon! You are right. It is corrected